spectrum-offchain-cardano/src/data/deposit.rs

```diff
@@ -71,6 +73,10 @@ where
                 let validator: DeployedValidator<{ StableFnPoolT2TDeposit as u8 }> = ctx.get();
                 validator.erased()
             }
+            OrderType::RoyaltyFn => {
```

Comment:
Please preserve proper naming, Royalty is not a function type, it's a feature of any pool

Reply:
I assume now it's just a RoyaltyConstFn, can be Royalty/BalanceStableFn in the future

Reply:
Done